src/tilesource.js

```diff
@@ -666,6 +670,27 @@ $.TileSource.prototype = {
         return {};
     },
 
+    /**
+     * A tile can have either context2D defined (TileSource.prototype.getContext2D)
+     * or its context2D is set manually. In those cases cache is not used and this function
+     * is irrelevant. Otherwise, the tile cache object is uniquely determined by this key:
+     * keys should be different if images are different!
+     * Note: default behaviour does not take into account post data.
+     * @param level tile level it was fetched with
```

Comment:
I forgot to mention this last time, but it would be good to add types to these param comments (`{String}`, etc.).